README.md

```diff
@@ -243,6 +243,37 @@ The string format expected by the framework is a string in the `route_identifier
 }
 ```
 
+## Flaggable Feature
```

Comment:
```suggestion
## Providing fallbacks for Features
```

What do you think? I was thinking that Flaggable might not be the best choice because you can disable features for other reasons, like IOS availability.